mteb/tasks/Classification/multilingual/AfriSentiLangClassification.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class AfriSentiLangClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="AfriSentiLangClassification",
+        description="AfriSentiLID is the largest LID classification dataset for African Languages.",
+        dataset={
+            "path": "HausaNLP/afrisenti-lid-data",
+            "revision": "f17cb5f3ec522ac604601fd09db9fd644ac66ca5",
+        },
+        reference="https://huggingface.co/datasets/HausaNLP/afrisenti-lid-data/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs={
+            "amh-Ethi",  # Amharic (Ethiopic script)
+            "arq-Arab",
+            "ary-Arab",  # Moroccan Arabic, Standard Arabic (Arabic script)
+            "hau-Latn",  # Hausa (Latin script), additional script if written in Ajami (Arabic script)
+            "ibo-Latn",  # Igbo (Latin script)
+            "kin-Latn",  # Kinyarwanda (Latin script)
+            "por-Latn",  # Portuguese (Latin script)
+            "pcm-Latn",  # Nigerian Pidgin (Latin script)
+            "swa-Latn",  # Swahili (macrolanguage) (Latin script)
+            "twi-Latn",  # Twi (Latin script)
+            "tso-Latn",  # Tsonga (Latin script)
+            "yor-Latn",  # Yoruba (Latin script)
+        },
```

Comment:
```suggestion
        eval_langs=[
            "amh-Ethi",  # Amharic (Ethiopic script)
            "arq-Arab",
            "ary-Arab",  # Moroccan Arabic, Standard Arabic (Arabic script)
            "hau-Latn",  # Hausa (Latin script), additional script if written in Ajami (Arabic script)
            "ibo-Latn",  # Igbo (Latin script)
            "kin-Latn",  # Kinyarwanda (Latin script)
            "por-Latn",  # Portuguese (Latin script)
            "pcm-Latn",  # Nigerian Pidgin (Latin script)
            "swa-Latn",  # Swahili (macrolanguage) (Latin script)
            "twi-Latn",  # Twi (Latin script)
            "tso-Latn",  # Tsonga (Latin script)
            "yor-Latn",  # Yoruba (Latin script)
        ],
```
This should be an array